### PR TITLE
Upload modul

### PR DESCRIPTION
**Pull request by Khegyi**

### CODE REVIEW

File: src/app.tsx [hunk 1]
```diff
@@ -32,58 +21,41 @@ export const useStyles = makeStyles(theme => ({
...
-    imgTitle: '',
-    imgDescription: '',
-    imgAuthor: '',
-    imgAuthorAvatar: '',
-    imgCreationDate: '',
-    imgSize: '',
-    imgDownloadUrl: '',
-  })
+  const classes = useStyles()
+  const [isNotificationShown, ShowNotification] = React.useState<boolean>(false)
```
Comment (B3zo0) [inline src/app.tsx]:
```suggestion
  const [isNotificationShown, setIsNotificationShown] = React.useState(false)
```
There is a convention in naming the variables destructed from useState. Please use that.

File: src/app.tsx [hunk 2]
```diff
@@ -32,58 +21,41 @@ export const useStyles = makeStyles(theme => ({
...
+  const [uploaddata, setUploaddata] = React.useState<boolean>(false)
```
Comment (B3zo0) [inline src/app.tsx]:
```suggestion
  const [isDataUploaded, setIsDataUploaded] = React.useState(false)
```
Please use camelCase for naming variables. Also it is a good pratice to prefix boolean variables with an _is_. This way you can see that this value is a boolean.

File: src/app.tsx [hunk 3]
```diff
@@ -32,58 +21,41 @@ export const useStyles = makeStyles(theme => ({
...
+
+  /** Display the notificationbar about successful upload
+   * @param {boolean} switcher Should the notification shown or not
+   */
+  function DisplayNotification(switcher: boolean) {
+    ShowNotification(switcher)
```
Comment (B3zo0) [inline src/app.tsx]: Honestly I don't see any value wraping this function with another function. Just pass it down as a prop.

File: src/app.tsx [hunk 4]
```diff
@@ -32,58 +21,41 @@ export const useStyles = makeStyles(theme => ({
...
-  function setUploaddataFunction() {
-    setUploaddata(true)
+  function CloseNotfication() {
+    ShowNotification(false)
   }
   /**
-   * Sets the Selected Image
-   * @param {number} imageIndex Seletected number's index.
+   * Sets the UploadData
+   * Determinate whatever an upload has accord or not
```
Comment (B3zo0) [inline src/app.tsx]:
```suggestion
   * Determines whether an upload has happened or not
```
accord = occured 😁

File: src/app.tsx [hunk 5]
```diff
@@ -32,58 +21,41 @@ export const useStyles = makeStyles(theme => ({
...
-      imgCreationDate: moment(new Date(selectedImage.CreationDate ? selectedImage.CreationDate : '')).format(
-        'YYYY-MM-DD HH:mm:ss',
-      ),
-      imgSize: `${(selectedImage.Size ? selectedImage.Size / 1024 / 1024 : 0).toFixed(2)} MB`,
-      imgDownloadUrl: selectedImage.Binary
-        ? repo.configuration.repositoryUrl + selectedImage.Binary.__mediaresource.media_src
-        : '',
-    })
+  function setUploaddataFunction() {
+    setUploaddata(true)
```
Comment (B3zo0) [inline src/app.tsx]: You should check the repositry events we have. With that you can subscribe to changes in repository like content created, modified, etc... Checkout https://github.com/SenseNet/sn-client/blob/develop/packages/sn-hooks-react/src/context/current-children.tsx

File: src/components/SimpleAppBar.tsx [hunk 6]
```diff
@@ -5,6 +5,7 @@ import { UploadControll } from './UploadControll'
 
 interface SimpleAppBarProps {
   uploadsetdata: () => void
+  notificationControll: (onoff: boolean) => void
```
Comment (B3zo0) [inline src/components/SimpleAppBar.tsx]:
```suggestion
  notificationControl: (onoff: boolean) => void
```

Comment (B3zo0) [reply src/components/SimpleAppBar.tsx]: also I would avoid naming that argument onoff. Use something like `isOpen`.

File: src/components/UploadControll.tsx [hunk 7]
```diff
@@ -34,35 +32,12 @@ const useStyles = makeStyles((theme: Theme) =>
...
-    if (reason === 'clickaway') {
-      return
-    }
-
-    setOpen(false)
-  }
-
   /**
    * Handle Uploaded File
    * @param e any
```
Comment (B3zo0) [inline src/components/UploadControll.tsx]: Do not use any in pickFile function.

File: README.md [hunk 8]
```diff
@@ -18,7 +18,7 @@ Be sure that you have installed a GIT client, Node.JS (latest or LTS). You can u
 
 ### Using as a template
```
Comment (B3zo0) [inline README.md]: This section doesn't make sense in your repository. It would be best to remove.

File: src/app.tsx (hunk 2 above)
Comment (Khegyi) [reply src/app.tsx]: done

File: src/app.tsx (hunk 1 above)
Comment (Khegyi) [reply src/app.tsx]: done

File: src/app.tsx (hunk 4 above)
Comment (Khegyi) [reply src/app.tsx]: done

File: src/components/SimpleAppBar.tsx (hunk 6 above)
Comment (Khegyi) [reply src/components/SimpleAppBar.tsx]: done

File: README.md (hunk 8 above)
Comment (Khegyi) [reply README.md]: done

File: src/components/UploadControll.tsx (hunk 7 above)
Comment (Khegyi) [reply src/components/UploadControll.tsx]: done

File: src/app.tsx (hunk 3 above)
Comment (Khegyi) [reply src/app.tsx]: done

**Pull request closed without merging**